include/datafile/UpgradeTo13.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * UpgradeTo1_3_0.h
+ *   Functor for upgrading data files to 1.3.0
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#pragma once
+
+#ifndef LMMS_UPGRADETO1_3_0_H
+#define LMMS_UPGRADETO1_3_0_H
+
+#include "datafile/DataFileUpgrade.h"
+
+#include <map>
+
+using StringMap = std::map<QString, QString>;
```

Comment:
I would move this out of global scope and inside `lmms::UpgradeTo_1_3_0`.

Reply:
Think I got this one, let me know if it's not quite right.